source/includes/fact-ssl-atlas-bic.rst

```diff
@@ -0,0 +1,4 @@
+.. note::
+
+   The |bi-atlas-short| uses TLS/SSL but does not require any TLS/SSL
+   settings configured in your system DSN.
```

Comment:
Something ungrammatical there. Could change to "...does not require any TLS/SSL settings to be configured in your system DSN", or "...does not require any TLS/SSL parameters in your system DSN".